x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/engine_creation.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import '../../../__mocks__/shallow_useeffect.mock';
+import { setMockValues } from '../../../__mocks__';
+
+import { EngineCreation } from './';
+
+describe('EngineCreation', () => {
+  const DEFAULT_VALUES = {
+    name: '',
+    rawName: '',
+    language: 'Universal',
+  };
+
+  describe('default values', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('renders', () => {
+      setMockValues(DEFAULT_VALUES);
```

Comment:
Since 90% of your tests are just using DEFAULT_VALUES, I'd strongly suggest moving it to the beforeEach:

```suggestion
      jest.clearAllMocks();
      setMockValues(DEFAULT_VALUES);
    });

    it('renders', () => {
```

In the test that doesn't use default values (the last one), you can just override it on a per-test basis by calling `setMockValues({ ...DEFAULT_VALUES, someOverride })` again at the start of the test